policy/modules/roles/auditadm.te

```diff
@@ -39,10 +39,6 @@ optional_policy(`
 	dmesg_exec(auditadm_t)
 ')
 
-optional_policy(`
-	dbus_role_template(auditadm, auditadm_r, auditadm_t)
```

Comment:
This and the below ones shouldn't be removed. The only templates that call `userdom_common_user_template()` is `userdom_unpriv_user_template()` and `userdom_admin_user_template()'.

Reply:
My mistake, auditadm and secadm call the unpriv user template, but the others don't. Please check with sediff to make sure no rules are lost.